CRITICAL - 2023-01-11 00:06:35 --> Undefined variable $id_user
in APPPATH\Views\pembimbing\divisi\assign_peserta.php on line 12.
 1 APPPATH\Views\pembimbing\divisi\assign_peserta.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $id_user', 'APPPATH\\Views\\pembimbing\\divisi\\assign_peserta.php', 12)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\pembimbing\\divisi\\assign_peserta.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1142): CodeIgniter\View\View->render('pembimbing/divisi/assign_peserta', [], true)
 5 APPPATH\Controllers\Pembimbing.php(177): view('pembimbing/divisi/assign_peserta', [...])
 6 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Pembimbing->setPeserta('19')
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pembimbing))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 00:12:56 --> Cannot use object of type stdClass as array
in APPPATH\Views\pembimbing\divisi\assign_peserta.php on line 19.
 1 SYSTEMPATH\View\View.php(213): include()
 2 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1142): CodeIgniter\View\View->render('pembimbing/divisi/assign_peserta', [], true)
 4 APPPATH\Controllers\Pembimbing.php(178): view('pembimbing/divisi/assign_peserta', [...])
 5 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Pembimbing->setPeserta('19')
 6 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pembimbing))
 7 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 00:13:20 --> Undefined property: stdClass::$name
in APPPATH\Views\pembimbing\divisi\assign_peserta.php on line 19.
 1 APPPATH\Views\pembimbing\divisi\assign_peserta.php(19): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$name', 'APPPATH\\Views\\pembimbing\\divisi\\assign_peserta.php', 19)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\pembimbing\\divisi\\assign_peserta.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1142): CodeIgniter\View\View->render('pembimbing/divisi/assign_peserta', [], true)
 5 APPPATH\Controllers\Pembimbing.php(178): view('pembimbing/divisi/assign_peserta', [...])
 6 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Pembimbing->setPeserta('19')
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pembimbing))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 00:14:11 --> Trying to access array offset on value of type null
in APPPATH\Controllers\Pembimbing.php on line 580.
 1 APPPATH\Controllers\Pembimbing.php(580): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Controllers\\Pembimbing.php', 580)
 2 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Pembimbing->setDivisi('19')
 3 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pembimbing))
 4 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 00:16:50 --> Cannot use object of type stdClass as array
in APPPATH\Views\pembimbing\divisi.php on line 43.
 1 SYSTEMPATH\View\View.php(213): include()
 2 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1142): CodeIgniter\View\View->render('pembimbing/divisi', [], true)
 4 APPPATH\Controllers\Pembimbing.php(155): view('pembimbing/divisi', [...])
 5 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Pembimbing->myDivision('Test 2', '113')
 6 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pembimbing))
 7 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 00:17:29 --> Cannot use object of type stdClass as array
in APPPATH\Views\pembimbing\divisi.php on line 43.
 1 SYSTEMPATH\View\View.php(213): include()
 2 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1142): CodeIgniter\View\View->render('pembimbing/divisi', [], true)
 4 APPPATH\Controllers\Pembimbing.php(155): view('pembimbing/divisi', [...])
 5 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Pembimbing->myDivision('Test 2', '113')
 6 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pembimbing))
 7 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 00:17:58 --> Cannot use object of type stdClass as array
in APPPATH\Views\pembimbing\divisi.php on line 47.
 1 SYSTEMPATH\View\View.php(213): include()
 2 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1142): CodeIgniter\View\View->render('pembimbing/divisi', [], true)
 4 APPPATH\Controllers\Pembimbing.php(155): view('pembimbing/divisi', [...])
 5 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Pembimbing->myDivision('Test 2', '113')
 6 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pembimbing))
 7 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 00:18:08 --> Undefined property: stdClass::$userId
in APPPATH\Views\pembimbing\divisi.php on line 47.
 1 APPPATH\Views\pembimbing\divisi.php(47): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$userId', 'APPPATH\\Views\\pembimbing\\divisi.php', 47)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\pembimbing\\divisi.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1142): CodeIgniter\View\View->render('pembimbing/divisi', [], true)
 5 APPPATH\Controllers\Pembimbing.php(155): view('pembimbing/divisi', [...])
 6 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Pembimbing->myDivision('Test 2', '113')
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pembimbing))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 00:18:21 --> syntax error, unexpected token ")"
in APPPATH\Controllers\Pembimbing.php on line 169.
 1 SYSTEMPATH\Autoloader\Autoloader.php(259): CodeIgniter\Autoloader\Autoloader->includeFile('APPPATH\\Controllers\\Pembimbing.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(237): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Pembimbing')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Pembimbing')
 4 SYSTEMPATH\CodeIgniter.php(852): class_exists('\\App\\Controllers\\Pembimbing', true)
 5 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 00:30:13 --> Undefined array key "statuspgj"
in APPPATH\Views\admin\datapengajuan.php on line 43.
 1 APPPATH\Views\admin\datapengajuan.php(43): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "statuspgj"', 'APPPATH\\Views\\admin\\datapengajuan.php', 43)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\admin\\datapengajuan.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1142): CodeIgniter\View\View->render('admin/datapengajuan.php', [], true)
 5 APPPATH\Controllers\Admin.php(379): view('admin/datapengajuan.php', [...])
 6 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Admin->dataPengajuan()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 00:38:45 --> Undefined array key "statuspgj"
in APPPATH\Views\spadmin\datapengajuan.php on line 41.
 1 APPPATH\Views\spadmin\datapengajuan.php(41): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "statuspgj"', 'APPPATH\\Views\\spadmin\\datapengajuan.php', 41)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\spadmin\\datapengajuan.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1142): CodeIgniter\View\View->render('spadmin/datapengajuan.php', [], true)
 5 APPPATH\Controllers\Notif.php(116): view('spadmin/datapengajuan.php')
 6 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Notif->datapengajuan('114')
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Notif))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
ERROR - 2023-01-11 21:57:51 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', 'root', 'simag', 3306, '', 0)
#1 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Model.php(772): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\simag-new-main\app\Controllers\Home.php(14): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\CodeIgniter.php(902): App\Controllers\Home->index()
#7 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\simag-new-main\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Model.php(772): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\simag-new-main\app\Controllers\Home.php(14): CodeIgniter\Model->__call('get', Array)
#5 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\CodeIgniter.php(902): App\Controllers\Home->index()
#6 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\simag-new-main\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2023-01-11 21:57:51 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
JOIN `info_peserta` ON `user`.`id`=`info_peserta`.`userId`
WHERE `info_peserta`.`endDate` > :info_peserta.endDate:', [...], false)
 3 SYSTEMPATH\Model.php(772): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Home.php(14): CodeIgniter\Model->__call('get', [])
 5 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
ERROR - 2023-01-11 23:06:01 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '', 'simag', 3306, '', 0)
#1 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Model.php(772): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\simag-new-main\app\Controllers\Home.php(14): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\CodeIgniter.php(902): App\Controllers\Home->index()
#7 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\simag-new-main\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Model.php(772): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\simag-new-main\app\Controllers\Home.php(14): CodeIgniter\Model->__call('get', Array)
#5 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\CodeIgniter.php(902): App\Controllers\Home->index()
#6 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\simag-new-main\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\simag-new-main\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2023-01-11 23:06:01 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
JOIN `info_peserta` ON `user`.`id`=`info_peserta`.`userId`
WHERE `info_peserta`.`endDate` > :info_peserta.endDate:', [...], false)
 3 SYSTEMPATH\Model.php(772): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Home.php(14): CodeIgniter\Model->__call('get', [])
 5 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 23:28:26 --> Call to undefined method App\Models\UserModel::getResult
in SYSTEMPATH\Model.php on line 774.
 1 APPPATH\Controllers\Pembimbing.php(437): CodeIgniter\Model->__call('getResult', [])
 2 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Pembimbing->dashboard()
 3 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pembimbing))
 4 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 23:34:47 --> Cannot use object of type stdClass as array
in APPPATH\Views\pembimbing\halamannilai.php on line 34.
 1 SYSTEMPATH\View\View.php(213): include()
 2 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1142): CodeIgniter\View\View->render('pembimbing/halamannilai.php', [], true)
 4 APPPATH\Controllers\Pembimbing.php(448): view('pembimbing/halamannilai.php')
 5 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Pembimbing->dashboard()
 6 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pembimbing))
 7 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
